src/coreclr-debug/activate.ts

```diff
@@ -106,50 +106,72 @@ function showDotnetToolsWarning(message: string): void {
     }
 }
 
-interface AdapterExecutableCommand {
-    command: string;
-}
+export class DebugAdapterDecriptor implements vscode.DebugAdapterDescriptorFactory {
```

Comment:
1. `DebugAdapterDecriptor` feels like a bad name to me. How about 'ClrDebugDescriptorFactory' or something like that?
2. Is there a reason we left this in activate.ts? Since it in a new class it feels better to put this in its own .ts file to me.